### DISCUSSION TITLE
Hierarchical clustering

### DISCUSSION THREAD

**Discussion by meyer-sp** [zqfang/GSEApy#286] [Q&A] [answered]
Hi,

first of all, GSEApy is awesome, I really like the simplicity and the beautiful plots it produces.

I am currently using the `dotplot` method to produce my visualizations. I would be interested in using hierarchical clustering, visualized by a dendrogram next to the dotplot to further group terms. In the documentation, I found the `y_order` attribute, which can be set to true to perform clustering. As I briefly checked the implementation, I noticed that `dendrogram` from `scipy.cluster.hierarchy` is already included. However, `dotplot`will not automatically draw the dendrogram tree, although `y_order` is set to `True`. Is there any known solution to this behavior?

Thanks a lot!

**Comment by zqfang** [comment, marked as answer]
The v1.1.4 adds the `ax` keyword to dotplot, which allows adding the dendrogram tree (and set `y_order` )

**Reply by meyer-sp** [reply]
Awesome, thanks a lot!
